cms/templates/export.html

```diff
@@ -6,18 +6,25 @@
   from django.utils.translation import ugettext as _
   import json
 %>
-<%block name="title">${_("Course Export")}</%block>
+<%block name="title">
+%if library:
```

Comment:
You can't just do "item" and "plural_item". In English we have only two plural forms; in languages such as Russian there are 6. You must use `ungettext` to translate plurals, always. I'm not really sure where the plural string(s) are that you're referring to where I can suggest how to properly do this.